Agro_easy/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
What purpose does this `manifest.json` serve? If you are not using it remove it from the repo